test/images/agnhost/mounttest/filePermissions.ps1

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Param(
+  [string]$FileName = $(throw "-FileName is required.")
+ )
+
+
+# read = read data | read attributes
+$READ_PERMISSIONS = 0x0001 -bor 0x0080
+
+# write = write data | append data | write attributes | write EA
+$WRITE_PERMISSIONS = 0x0002 -bor 0x0004 -bor 0x0100 -bor  0x0010
+
+# execute = read data | file execute
+$EXECUTE_PERMISSIONS = 0x0001 -bor 0x0020
+
+
+function GetFilePermissions($path) {
```

Comment:
I lack sufficient windows expertise to really review this; can we get someone else to take a look at this file?

Reply:
FWIW, it's what it has been used in tests like ``should be consumable from pods in volume with mappings`` since about 1 year. You can see it here: https://testgrid.k8s.io/sig-windows#aks-engine-azure-windows-master